### PR TITLE
feat: [SRM-11246]: Template bar in MS  input set

### PR DESCRIPTION
**Pull request by deepesh-ui**
### Summary. 

- Template bar in Monitored Service InputSet
- Remove use template button for MS template

#### Screenshots

Screenshot 2022-07-28 at 8 52 24 AM

Screenshot 2022-07-28 at 9 46 22 PM

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Review by praneshg239 — DISMISSED** [review]

**Comment by deepesh-ui** [comment]
retrigger titlecheck

**Comment by deepesh-ui** [comment]
retrigger titlecheck

File: src/modules/85-cv/pages/monitored-service/MonitoredServiceInputSetsTemplate/MonitoredServiceInputSetsTemplate.tsx [hunk 1]
```diff
@@ -141,6 +145,28 @@ export default function MonitoredServiceInputSetsTemplate({
       })
   }
 
+  const onUseTemplate = async (): Promise<void> => {
+    const { template } = await getTemplate({ templateType: 'MonitoredService' })
```
Comment (harshilbits) [inline src/modules/85-cv/pages/monitored-service/MonitoredServiceInputSetsTemplate/MonitoredServiceInputSetsTemplate.tsx:149]: Please move MonitoredService to templateType enum

File: src/modules/85-cv/pages/monitored-service/MonitoredServiceInputSetsTemplate/MonitoredServiceInputSetsTemplate.tsx [hunk 2]
```diff
@@ -190,6 +216,13 @@ export default function MonitoredServiceInputSetsTemplate({
           return (
             <>
               <Layout.Vertical>
+                <TemplateBar
+                  templateLinkConfig={{
+                    templateRef: templateRefData.identifier as string,
```
Comment (harshilbits) [inline src/modules/85-cv/pages/monitored-service/MonitoredServiceInputSetsTemplate/MonitoredServiceInputSetsTemplate.tsx:221]: what if templateRefData is null, good to have a null check or default values

**Review by harshilbits — CHANGES_REQUESTED** [review]

**Review by harshilbits — APPROVED** [review]

**Review by yc147 — DISMISSED** [review]

**Review by yc147 — CHANGES_REQUESTED** [review]
Let’s explore the possibility of doing it through RBAC

**Pull request closed without merging**